### PR TITLE
Update/link to the api docs

### PR DESCRIPTION
**Pull request by amychen1776**
## Description & motivation
This PR creates a landing page for trying to find information about the dbt Cloud API on the docs site. This includes how to find the tokens in dbt cloud as well as link to the information about the endpoints. 

## To-do before merge
Include any notes about things that need to happen before this PR is merged, e.g.:
- [ ] Check to make sure that this is available to team plans

## Pre-release docs
Is this change related to an unreleased version of dbt?
- [ ] Yes: please
    - update the base branch to `next`
    - add Changelog components: `<Changelog>[New/Changed] in v0.x.0</Changelog>`
    - add links to the "New and changed documentation" section of the latest [Migration Guide](../website/docs/docs/guides/migration-guide)
- [x ] No: please ensure the base branch is `current`
- [ ] Unsure: we'll let you know!

## Checklist
If you added new pages (delete if not applicable):
- [x ] The page has been added to `website/sidebars.js`
- [x ] The new page has a unique filename

### CODE REVIEW

File: website/docs/docs/dbt-cloud/cloud-api.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+If you are on the `Team` or `Enterprise` dbt Cloud plan, you will have access to the 
+dbt Cloud API.
+
+Common uses for our API would be:
+
+- Downloading artifacts after a job has completed 
+- Kicking off a job run 
+- Creating/modifying/deleting objects in dbt Cloud 
+
+Our API endpoints can be found [here](dbt-cloud/api/).
```
Comment (clrcrl) [inline website/docs/docs/dbt-cloud/cloud-api.md:17]: I think you want:
```suggestion
Our API endpoints can be found [here](dbt-cloud/api).
```

Comment (clrcrl) [reply website/docs/docs/dbt-cloud/cloud-api.md:17]: Nope that's not either :)

Comment (clrcrl) [inline website/docs/docs/dbt-cloud/cloud-api.md]: @drewbanin — I'm actually not sure how to get this one to work!

Comment (clrcrl) [reply website/docs/docs/dbt-cloud/cloud-api.md]: Turns out you want:
```suggestion
Our API endpoints can be found [here](/dbt-cloud/api).
```

File: website/docs/docs/dbt-cloud/cloud-api.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+### Overview
+
+If you are on the `Team` or `Enterprise` dbt Cloud plan, you will have access to the 
+dbt Cloud API.
+
+Common uses for our API would be:
+
+- Downloading artifacts after a job has completed 
+- Kicking off a job run 
+- Creating/modifying/deleting objects in dbt Cloud
```
Comment (drewbanin) [inline website/docs/docs/dbt-cloud/cloud-api.md]:
```suggestion
The dbt Cloud API can be used to:

- Download artifacts after a job has completed 
- Kick off a job run from an orchestration tool
- Manage your dbt Cloud account
```

File: website/docs/docs/dbt-cloud/cloud-api.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+If you are on the `Team` or `Enterprise` dbt Cloud plan, you will have access to the 
+dbt Cloud API.
+
+Common uses for our API would be:
+
+- Downloading artifacts after a job has completed 
+- Kicking off a job run 
+- Creating/modifying/deleting objects in dbt Cloud 
+
+Our API endpoints can be found [here](/dbt-cloud/api).
```
Comment (drewbanin) [inline website/docs/docs/dbt-cloud/cloud-api.md]:
```suggestion
Full documentation for the dbt Cloud API can be found [here](/dbt-cloud/api).
```

File: website/docs/docs/dbt-cloud/cloud-api.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+You can find your API token in your user's profile under `API Access`.  
+
+<Lightbox src="/img/api-access-profile.png" title="API Access" />
+
+#### Service API token
+To enable this for your account, you will need to contact Support via Intercom 
+or support@getdbt.com. Once it is enabled, you can create these tokens by going to 
+Account Settings > Service Tokens.
+
+<LoomVideo id="/55eb660a52bf4417b1274c6a55796cda" />
```
Comment (drewbanin) [inline website/docs/docs/dbt-cloud/cloud-api.md]: I don't think this works - you might need to remove the leading slash? See the deploy preview here: https://deploy-preview-544--docs-getdbt-com.netlify.app/docs/dbt-cloud/cloud-api

File: website/docs/docs/dbt-cloud/cloud-api.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+#### User API tokens
+You can find your API token in your user's profile under `API Access`.  
+
+<Lightbox src="/img/api-access-profile.png" title="API Access" />
+
+#### Service API token
+To enable this for your account, you will need to contact Support via Intercom 
+or support@getdbt.com. Once it is enabled, you can create these tokens by going to 
+Account Settings > Service Tokens.
```
Comment (drewbanin) [inline website/docs/docs/dbt-cloud/cloud-api.md]:
```suggestion
Service Account API tokens are currently available in a beta for dbt Cloud Enterprise customers. To request access to the Service Account API token preview, please contact support in-app or by emailing support@getdbt.com.

Once enabled, you can manage Service Account API Tokens from the Account Settings > Service Tokens page within dbt Cloud.
```

File: website/docs/docs/dbt-cloud/cloud-api.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+Our API endpoints can be found [here](/dbt-cloud/api).
+
+### How to Access the API 
+
+#### User API tokens
+You can find your API token in your user's profile under `API Access`.  
+
+<Lightbox src="/img/api-access-profile.png" title="API Access" />
+
+#### Service API token
```
Comment (drewbanin) [inline website/docs/docs/dbt-cloud/cloud-api.md]:
```suggestion
#### Service Account API tokens
```

**Review by drewbanin — COMMENTED** [review]
Looking good! I dropped some comments/suggestions in here - mostly looking to match verb tense with the rest of the Cloud docs + address the Loom embed. Let me know if you have any thoughts or comments!

**Comment by amychen1776** [comment]
@drewbanin Thank you so much for your suggestions! I just pushed the updates if you don't mind taking a look again. 

Edit: Oh no, I just realized that the deploy failed so going to check on that.

**Comment by drewbanin** [comment]
@amychen1776 i think this was some kind of netlify blip - looks like it was unable to check out the repo? Re-running the staging deploy now :)

File: website/docs/docs/dbt-cloud/cloud-api.md (hunk 2 above)
Comment (drewbanin) [reply website/docs/docs/dbt-cloud/cloud-api.md]: huh - I see this one market as resolved, but the change wasn't actually applied. My thinking here is just that we shouldn't call this "our" API, we should call it "the" API :)

File: website/docs/docs/dbt-cloud/cloud-api.md [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+If you are on the `Team` or `Enterprise` dbt Cloud plan, you will have access to the 
+dbt Cloud API.
+
+Common uses for our API would be:
+
+- Downloading artifacts after a job has completed 
+- Kicking off a job run 
+- Creating/modifying/deleting objects in dbt Cloud 
+
+Our API endpoints can be found [here](/
```
Comment (drewbanin) [inline website/docs/docs/dbt-cloud/cloud-api.md]: and same here - it's not _our_ API!

**Comment by amychen1776** [comment]
@drewbanin I think I accidentally overwrote some of my commits for your suggestions when I changed the Loom link locally. Checking the deploy preview now, I think everything has been fixed

**Review by drewbanin — APPROVED** [review]
Thanks Amy - looks great!

**Pull request merged**